WalletWasabi.Gui/Tabs/WalletManager/GenerateWallets/GenerateWalletSuccessView.xaml

```diff
@@ -3,13 +3,27 @@
              xmlns:controls="clr-namespace:WalletWasabi.Gui.Controls;assembly=WalletWasabi.Gui"
              x:Class="WalletWasabi.Gui.Tabs.WalletManager.GenerateWallets.GenerateWalletSuccessView">
   <StackPanel Margin="20" Spacing="10">
-    <TextBlock Text="Write down these Recovery Words!" FontWeight="Bold" />
-    <Border Padding="2" Background="White" HorizontalAlignment="Stretch" Opacity="0.8">
-      <controls:ExtendedTextBox Text="{Binding MnemonicWords, Mode=TwoWay}" Background="Transparent" FontWeight="Bold" Foreground="Black" HorizontalAlignment="Center" Classes="selectableTextBlock" />
+    <TextBlock Text="Write down these Recovery Words &amp; Your Password" FontWeight="Bold" />
+    <Border Padding="20" Margin="20 0" HorizontalAlignment="Center">
+      <ItemsControl Items="{Binding MnemonicWords}">
+        <ItemsControl.Styles>
+          <Style Selector="TextBlock">            
+            <Setter Property="FontSize" Value="22" />
+            <Setter Property="Margin" Value="40 10" />
+            <Setter Property="Height" Value="32" />
+          </Style>
+        </ItemsControl.Styles>
+        <ItemsControl.ItemsPanel>
+          <ItemsPanelTemplate>
+            <WrapPanel MaxHeight="220" Orientation="Vertical" />
+          </ItemsPanelTemplate>
+        </ItemsControl.ItemsPanel>
+      </ItemsControl>
     </Border>
     <TextBlock Text="You can recover your wallet on any computer with" />
     <TextBlock Text="- your Recovery Words AND" />
     <TextBlock Text="- your Password." />
-    <Button Margin="0 10 0 0" Background="IndianRed" Content="I wrote down my Recovery Words!" Command="{Binding ConfirmCommand}" />
+    <CheckBox Margin="0 20 0 0" HorizontalAlignment="Center" IsChecked="{Binding IsConfirmed}" Content="I have written down my password and recovery words." />
+    <Button Margin="0 10 0 0" HorizontalAlignment="Center" Content="Continue" Command="{Binding ConfirmCommand}" />
```

Comment:
Maybe the button should say `Generate` instead of `Continue`?

Reply:
done